services/supervisord/supervisord_test.go

```diff
@@ -56,6 +58,45 @@ func TestConfig(t *testing.T) {
 	}
 }
 
+func TestDBaaSController(t *testing.T) {
+	t.Parallel()
+
+	pmmUpdateCheck := NewPMMUpdateChecker(logrus.WithField("component", "supervisord/pmm-update-checker_logs"))
+	configDir := filepath.Join("..", "..", "testdata", "supervisord.d")
+	s := New(configDir, pmmUpdateCheck)
+
+	tests := []bool{
+		true,
+		false,
+	}
+	var tp *template.Template
+	for _, tmpl := range templates.Templates() {
+		if tmpl.Name() == "dbaas-controller" {
+			tp = tmpl
+			break
+		}
+	}
+
+	for _, enabled := range tests {
+		st := models.Settings{
+			DBaaS: struct {
+				Enabled bool `json:"enabled"`
+			}{
+				Enabled: enabled,
+			},
+		}
+		if !st.DBaaS.Enabled {
```

Comment:
What's the reason to add `false` case and skip it?

Reply:
Added expected result for disabled into another file.